### PR TITLE
Improve notifier

### PR DESCRIPTION
**Pull request by jreidinger**
this change fixed issue #55 and issue #69

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -18,8 +18,8 @@
     "ngEmbed": "git://github.com/ritz078/ngEmbed.git#a7048132d3cffd0ca39dec9e851762cb34102bb9",
     "angular-hotkeys": "chieffancypants/angular-hotkeys#~1.4.5",
     "angular-gridster": "~0.13.5",
-    "angular-toastr": "~1.4.1",
-    "angular-audio": "~1.7.0",
+    "angular-extended-notifications": "~1.0.4",
+    "angular-audio": "~1.7.0"
```
Comment (ancorgs) [inline bower.json]: comma is missing

File: src/app/components/notifier/notifier.service.js [hunk 2]
```diff
@@ -9,19 +9,35 @@
   'use strict';
 
   angular.module('janusHangouts')
-    .service('Notifier',  Notifier);
+    .service('MuteNotifier',  MuteNotifier);
```
Comment (ancorgs) [inline src/app/components/notifier/notifier.service.js]: please rename the file as well

File: src/app/components/notifier/notifier.service.js [hunk 3]
```diff
@@ -9,19 +9,35 @@
...
 
-  Notifier.$inject = ['$animate', 'toastr', 'ngAudio'];
+  MuteNotifier.$inject = ['$animate', 'notifications', 'ngAudio',
+    'ActionService'];
```
Comment (ancorgs) [inline src/app/components/notifier/notifier.service.js]: No need for a line break here. We don't stick to 80 columns in general.

File: src/app/components/videochat/jh-video-chat.html [hunk 4]
```diff
@@ -19,7 +19,7 @@
   </header>
 </div>
 
-<div class="row">
+<div class="row" id="body">
```
Comment (ancorgs) [inline src/app/components/videochat/jh-video-chat.html]: Not sure if I like "body" as an id for something that is actually just part of a directive. Sounds like the body of the full page.

File: src/app/index.js [hunk 5]
```diff
@@ -9,7 +9,8 @@
 
 angular.module('janusHangouts', ['ngAnimate', 'ngCookies', 'ngTouch',
                'ngSanitize', 'blockUI', 'ui.router', 'ui.bootstrap', 'ngEmbed',
-               'janusHangouts.config', 'cfp.hotkeys', 'gridster', 'toastr', 'ngAudio',
+               'janusHangouts.config', 'cfp.hotkeys', 'gridster',
+               'toastr', 'ngAudio', 'angular-extended-notifications',
```
Comment (ancorgs) [inline src/app/index.js]: remove toastr from here

File: src/app/components/notifier/notifier.service.js [hunk 6]
```diff
@@ -9,19 +9,35 @@
...
     this.info = info;
     var bell = ngAudio.load("assets/sounds/bell.ogg");
+    var noShow = {};
 
     function info(text) {
-      bell.play();
-      toastr.info(text, {
-        onShown: function() { bell.play(); },
-        timeOut: 3000
+      if (text in noShow)
```
Comment (ancorgs) [inline src/app/components/notifier/notifier.service.js]: Mmm, now that this notifier is not generic anymore, but only related to muting messages, I would rather move the real strings inside the notifier providing functions like
```
MuteNotifier.speaking()
MuteNotifier.muted()
```

File: bower.json (hunk 1 above)
Comment (jreidinger) [inline bower.json]: thanks for catching merge artifact

File: src/app/components/notifier/notifier.service.js (hunk 2 above)
Comment (jreidinger) [inline src/app/components/notifier/notifier.service.js]: ah, my bad memory, will fix it

File: src/app/components/notifier/notifier.service.js (hunk 3 above)
Comment (jreidinger) [inline src/app/components/notifier/notifier.service.js]: yeah, I was curious, so is there any style guide?

File: src/app/components/notifier/notifier.service.js (hunk 6 above)
Comment (jreidinger) [inline src/app/components/notifier/notifier.service.js]: ok, make sense for me.

File: src/app/components/videochat/jh-video-chat.html (hunk 4 above)
Comment (jreidinger) [inline src/app/components/videochat/jh-video-chat.html]: hmm, there is header above and footer below, so for me body is native name from old school pure html times :)

Comment (jreidinger) [inline src/app/components/videochat/jh-video-chat.html]: but I am open to any name, just need it for attaching notify

File: src/app/index.js (hunk 5 above)
Comment (jreidinger) [inline src/app/index.js]: ugh, another merge artifact, I am quite bad in merging conflicts :(

File: src/app/components/videochat/jh-video-chat.html (hunk 4 above)
Comment (ancorgs) [inline src/app/components/videochat/jh-video-chat.html]: Ids are unique across the DOM. Let's use "videochat-body"

Comment (jreidinger) [inline src/app/components/videochat/jh-video-chat.html]: ok, make sense

**Comment by ancorgs** [comment]
LGTM

**Pull request merged**